helpdesk/overview.rst

```diff
@@ -6,4 +6,5 @@ Overview
    :titlesonly:
 
    overview/getting_started
-   overview/sla
\ No newline at end of file
+   overview/sla
+   overview/receiving_ticketsgit
```

Comment:
@jubodoo @Lau-Odoo @kbr-odoo Don't forget to build the doc before merging/reviewing/pushing !
This will be fixed soon but, if undetected, build errors like this can freeze the update of the doc until they get fixed ;)

Reply:
I am aware of it. I always build. And yeah, it will be fixed. Thanks!

Reply:
I came here to tell there was an issue ^^'
Good luck :-)